tools/nnyeah/nnyeah/nnyeah.csproj

```
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net$(BundledNETCoreAppTargetFrameworkVersion)</TargetFramework>
```

should we do the same like we did for mlaunch?
```suggestion
    <TargetFramework>net$(BundledNETCoreAppTargetFrameworkVersion)</TargetFramework>
    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
```

---

It's a bit simpler in xamarin-macios, because we already have a variable we can use everywhere.